src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/BlobDispatcherTask.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.blobrouter.tasks;
+
+import com.azure.storage.blob.BlobServiceAsyncClient;
+import com.azure.storage.blob.models.BlobContainerItem;
+import org.slf4j.Logger;
+import uk.gov.hmcts.reform.blobrouter.config.ServiceConfiguration;
+import uk.gov.hmcts.reform.blobrouter.tasks.processors.ContainerProcessor;
+import uk.gov.hmcts.reform.blobrouter.util.StorageClientManager;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class BlobDispatcherTask {
+
+    static final String TASK_NAME = "blob-dispatcher";
```

Comment:
access modifier missing
and do we really need this, without this logging may be faster ?

Reply:
This variable is being used in the test. It may be a tiny bit faster but we don't have to hard code the task name in the verifications